### PR TITLE
Added version flag

### PR DESCRIPTION
**Pull request by AlexanderDavid**
Fix issue #14 by adding a version flag. Version number is generated by the pkg_resources library (included in setuptools).

### CODE REVIEW

**Comment by plant99** [comment]
Thanks for taking this up, Alexander.

Following are some small details which needs to be taken care of

1. Do a rebase with this repository's master branch. I see some readme changes aren't reflected in your fork.
2. Please set the destination branch of the PR as `develop`. That's where all the feature changes should go before next release.

After this check very minor suggestions in `felicette/cli.py`

File: felicette/cli.py [hunk 1]
```diff
@@ -47,8 +48,18 @@ def trigger_download_and_processing(landsat_item, bands):
...
+    "--version",
+    default=False,
+    is_flag=True,
+    help="Show the version number and quit",
+)
+def main(coordinates, location_name, pan_enhancement, no_preview, vegetation, version):
     """Satellite imagery for dummies."""
+    if version:
+        version_no = pkg_resources.require("felicette")[0].version
+        exit_cli(print, f"felicette -- Satallite imagery for dummies. Version {version_no}.")
```
Comment (plant99) [inline felicette/cli.py:62]: In most of the CLI tools the version description is of the format `<package name> <version number> <date[optional]>`
Let's remove the description "Satellite imagery for dummies. Version", and keep `"felicette {version_no}"`

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Satellite imagery for dummies.
```
Comment (plant99) [inline README.md:8]: Please rebase these changes with current `develop`

File: felicette/cli.py (hunk 1 above)
Comment (AlexanderDavid) [reply felicette/cli.py]: Removed

**Comment by AlexanderDavid** [comment]
I had some issues with the rebase but I think I got it! Thanks for the feedback @plant99

**Comment by plant99** [comment]
Now the code/doc changes looks completely good to me @AlexanderDavid . 🥳 

Now there are 5 commits, but now you see bf9648a0e9fdab770f4cdb92f5567d5e223ff017 and 9d17c912b8d7424dc166e85cef7f67124f7de9c3 are nearly same. 

What we ideally want is for all the commit to make meaningful changes, right?

Could you please do something like

```
git branch version-b
git checkout version-b
git reset HEAD~5

# now you have the current state of code but no commits, so commit both of the files
git commit felicette/cli.py -m "add version flag"
git commit README.md -m "add to doc"
```

I hope my idea looks presentable.

**Comment by AlexanderDavid** [comment]
Commits are now meaningful

**Pull request merged**